components/intake/excitement.jsx

```diff
@@ -7,6 +7,7 @@ const Excitement = ({onClick}) => {
   const company = 'Florish';
   const context = useContext(Context);
   const handleClick = (excitement) => {
+    excitement = options.findIndex(element => element.toLowerCase() === excitement) + 1;
```

Comment:
given we're using this functionality in 2 files, can we transform it into a reusable function?